test/integration/integration_test.go

```diff
@@ -102,6 +113,25 @@ func (i *integrationTest) setup() {
 		}
 		i.setupComplete = true
 	}
+
+	from := flag.CommandLine
+	if vflag := from.Lookup("v"); vflag != nil {
+		fmt.Fprint(os.Stdout, "integration_test.go: found -v flag from starter bash script with initial glog level ", getLogLevel(), "\n")
```

Comment:
Somehow, I prefer `Fprintf` instead of `Fprint`, in that case the output seems to be more readable imho.